### PR TITLE
Support for searching multiple dependencies

### PR DESCRIPTION
**Pull request by sgleske-ias**
Extend the search to include sprint4shell in addition to log4shell.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,7 +38,10 @@ docker:
 	docker build --build-arg MAVEN_VERSION="$(MAVEN_VERSION)" -t deep-scanner .
 
 archive:
-	find . -maxdepth 2 -type f -name '*.giturl' | cut -d/ -f2 | sort -u | xargs -- tar -czf results.tar.gz
+	( \
+	find . -maxdepth 2 -type f -name '*.giturl' | cut -d/ -f2; \
+	find * -maxdepth 0 -type d | xargs -n1 -I{} echo {}.txt | xargs -n1 -I{} -- find * -maxdepth 1 -type f -name {}; \
+	) | sort -u | xargs -- tar -czf results.tar.gz
```
Comment (sgleske-ias) [inline Makefile:44]: This is useful to capture active repository list when creating an archive.  Active repository list is generated by running

```bash
export GITHUB_TOKEN
read -ersp token:  GITHUB_TOKEN
./get-unarchived-repositories.sh
```

File: Makefile [hunk 2]
```diff
@@ -57,7 +60,7 @@ progress:
 	@docker ps -aq | wc -l
 
 csv:
-	./generate-csv.sh * > log4j-core-versions.csv
+	./generate-csv.sh * > scanned-versions.csv
```
Comment (sgleske-ias) [inline Makefile:63]: This now scans for more than log4j.

File: README.md [hunk 3]
```diff
@@ -293,6 +293,12 @@ build tools then you may want to also exclude them in the following command.
 See `java -jar cloneable.jar --help` for additional usage and exclusion
 capabilities.
 
+# Roadmap
+
+- [x] Initial release for log4j-scans
+- [x] spring4shell release for scanning multiple dependencies.
+- [ ] Support for more languages relying on CycloneDX BOM format.
```
Comment (sgleske-ias) [inline README.md:300]: CycloneDX BOM is the way of the future.  This repository was thrown together in a time of need but a proper scanner would read the softwared BOM (bill of materials) to find vulnerable dependencies.

**Review by tplass-ias — APPROVED** [review]

**Comment by sgleske-ias** [comment]
Great catch on not using `checkDependencyFormat` @tplass-ias .  I updated the code to use it and found/fixed a bug when it was used.

**Pull request merged**